common/src/components/NotificationsList.tsx

```diff
@@ -0,0 +1,167 @@
+  right: var(--spacing-m);
+  transform: translateY(-50%);
+  padding: var(--spacing-xs);
+  padding-right: 0;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  &:hover {
+    color: var(--color-black-50);
+  }
+`;
+
+const NotificationsListItem = ({
+  notification,
+  closeFn,
+}: NotificationItemProps) => {
+  const displayDate = new Date(notification.activeFrom || "");
+  let notificationType;
+  switch (notification.level) {
+    case "EXCEPTION":
+      notificationType = "alert" as NotificationType;
+      break;
+    case "WARNING":
+      notificationType = "error" as NotificationType;
+      break;
+    default:
+      notificationType = "info" as NotificationType;
+  }
+  const handleCloseButtonClick = () => {
+    closeFn((prev) => {
+      if (!prev?.length) return [notification.id];
+      return [...prev, notification.id];
+    });
+  };
+  return (
+    <NotificationBackground
+      style={{ borderBottom: `1px solid var(--color-${notificationType}-dark` }}
+    >
+      <NotificationContainer type={notificationType}>
+        {notification.activeFrom && (
+          <NotificationDate>{`${displayDate.getDate()}.${displayDate.getMonth()}`}</NotificationDate>
+        )}
+        <NotificationText>
+          {getTranslation(notification, "message")}
+        </NotificationText>
+        <CloseButton onClick={() => handleCloseButtonClick()}>
+          <IconCross size="s" />
+        </CloseButton>
+      </NotificationContainer>
+    </NotificationBackground>
+  );
+};
+
+const NotificationsList = ({ target }: NotificationListProps) => {
+  const { data: notificationData } = useQuery<Query>(BANNER_NOTIFICATIONS_LIST);
+
+  const notificationsList = notificationData?.bannerNotifications?.edges.map(
+    (edge) => edge?.node
+  );
+
+  const [closedNotificationsList, setClosedNotificationsList] = useLocalStorage<
+    string[]
+  >("seenNotificationsList", []);
+  const maximumNotificationAmount = 2;
+
+  // Separate notifications by level
+  const errorNotificationsList = notificationsList?.filter(
+    (item) => item?.level === "WARNING"
+  );
+  const alertNotificationsList = notificationsList?.filter(
+    (item) => item?.level === "EXCEPTION"
+  );
+  const infoNotificationsList = notificationsList?.filter(
+    (item) => item?.level === "NORMAL"
+  );
+  // Merge grouped notifications prioritised by level
+  const groupedNotificationsList = [
+    ...(errorNotificationsList || []),
+    ...(alertNotificationsList || []),
+    ...(infoNotificationsList || []),
+  ];
+  // Filter out notifications that have been closed by the user, or aren't targeted to the user
+  const displayedNotificationsList = groupedNotificationsList?.filter(
+    (item) => {
+      if (
+        !closedNotificationsList ||
+        closedNotificationsList.includes(String(item?.id)) ||
+        (item?.target !== target && item?.target !== "ALL")
+      ) {
+        return false;
+      }
+      return true;
```

Comment:
`return (!closedNotif...)`? or with just: `(item => (!closedNotif...)`

Reply:
Haha! *facepalm* Indeed.. this became the much more concise:
`(item) =>
      !(closedNotificationsList as string[]).includes(String(item?.id)) ||
      !(item?.target !== target && item?.target !== "ALL")`